pkg/controller.go

```diff
@@ -263,13 +262,3 @@ func createControllerContainers(csi *csibaremetalv1.Deployment) []corev1.Contain
 		},
 	}
 }
-
-func createControllerSecurityContext(ctx *components.SecurityContext) *corev1.PodSecurityContext {
```

Comment:
please don't remove this func here, otherwise the csi controller security context would be made unconfigurable. It would be better to change this func a little to fix this issue.

Reply:
Got it, thanksss for reminding.